Source/EasyNetQ/EasyNetQ.csproj

```diff
@@ -1,22 +1,32 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>A nice .NET API for RabbitMQ</Description>
-    <TargetFrameworks>netstandard2.0;net8.0</TargetFrameworks>
-    <Nullable>enable</Nullable>
+    <TargetFrameworks>netstandard2.0;net8.0;net9.0</TargetFrameworks>
```

Comment:
Should we consider adding `net10.0` support since `net10.0` is the latest LTS version?